ci-operator/config/openshift/contour-operator/openshift-contour-operator-master.yaml

```diff
@@ -0,0 +1,47 @@
+base_images:
+  base:
+    name: "4.9"
+    namespace: ocp
+    tag: base
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.16
+promotion:
+  name: "4.9"
+  namespace: ocp
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  name: "4.9"
+  namespace: ocp
+tests:
+- as: e2e-aws
+  steps:
+    cluster_profile: aws
+    workflow: openshift-e2e-aws
+- as: test
+  commands: make test
+  container:
+    from: src
+- as: test-e2e
```

Comment:
whats the difference between the `e2e-aws` job and the `test-e2e` job?

Reply:
I will remove the section for e2e-aws